java/image.bzl

```diff
@@ -180,13 +182,23 @@ def _jar_app_layer_impl(ctx):
         for f in unavailable + [classpath_file]
     }
 
+    # legacy_run_behavior and docker_run_flags from base override those from
```

Comment:
I dont like this code being duplicated so much. This is just accessing ctx, and passing values to call of  _container. Cant we just do all this in container/image.bzl

Reply:
Good point. moved to container/image.bzl